tests/commands/checksec.py

```diff
@@ -4,7 +4,7 @@
 
 from tests.utils import (
     gdb_run_cmd,
-    _target,
+    debug_target,
```

Comment:
sorry to make this more than a search and replace, but can you sort these alphabetically? Trying to get to a place where we can turn on a bunch of linters.

Reply:
i feel we should make a new PR for the sorting change, because there will be many variations of things to sort for enabling linters, e.g.:
![image](https://github.com/hugsy/gef/assets/1533278/3664ff75-9b22-4a47-a289-f4133c77c027)

we can probably enable the linters with certain checks disabled, then follow up with changes to gradually enable certain checks, such as sorted imports. and i can help with that.